### PR TITLE
feature(DSG-2332): Employer portal landing page

### PR DESCRIPTION
**Pull request by thechechogomez**
# Basics

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Styling and layout changes
- [ ] Code style changes (formatting, local variables)
- [ ] Refactoring (no functional changes)
- [ ] Build or CI related changes
- [ ] Tests
- [ ] Other... Please describe:

## What changes does this PR introduce?

Ticket: [DSG-2332](https://dol-ewf.atlassian.net/browse/DSG-2332)
Figma: [WI-DWD](https://www.figma.com/file/lRexK80Jk7cdvCKemUUzOG/WI-DWD-UI?type=design&node-id=969-98599&mode=design&t=wXFaEoYyo5glvUKc-0)

It creates a landing page for the Employer portal and changes the style of the admin landing page to the match the employer page. 

## Screenshot of the affected changes

Screenshot 2023-11-22 at 3 56 34 PM

Screenshot 2023-11-22 at 3 58 17 PM

## Checklist

- [ ] I have updated the documentation accordingly
- [X] I have confirmed all acceptance criteria defined by the story have been met
- [x] I have confirmed that Sonar Qube findings for new code are reviewed and fixed, or will be whitelisted

[DSG-2332]: https://dol-ewf.atlassian.net/browse/DSG-2332?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by jgnuvalence — APPROVED** [review]
this looks good to me, I like the new Nuverial dashboard card and card group components for reusability

File: libs/employer/feature/dashboard/src/lib/models/dashboard-configuration.model.ts [hunk 1]
```diff
@@ -26,5 +26,54 @@ export const DashboardConfiguration: IDashboardConfiguration = {
         { key: 'first-quarter-deferral', label: 'First Quarter Deferral' },
       ],
     },
+    {
+      description: 'View and update account information and status, and view election information',
+      icon: 'shield_person',
+      name: 'View & Manage Account',
+      route: 'view-manage-account',
+      subCategories: [],
```
Comment (derekdiep42) [inline libs/employer/feature/dashboard/src/lib/models/dashboard-configuration.model.ts:34]: Did you check out the designs? Can we will in these for any of these other tiles?

Comment (thechechogomez) [reply libs/employer/feature/dashboard/src/lib/models/dashboard-configuration.model.ts:34]: Didn't see them. I'll fill them in

File: libs/employer/feature/dashboard/src/lib/components/dashboard/dashboard.component.ts [hunk 2]
```diff
@@ -1,79 +1,22 @@
...
+  imports: [NuverialDashboardCardComponent, NuverialDashboardCardsGroupComponent, CommonModule],
   selector: 'dsg-dashboard',
   standalone: true,
   styleUrls: ['./dashboard.component.scss'],
   templateUrl: './dashboard.component.html',
 })
 export class DashboardComponent {
-  public profile$: Observable<UserModel | null> = this._profileService.getProfile$();
-  public transactions$: Observable<TransactionModel[]> = of([]);
+  public readonly cards = of(this._dashboardService.getDashboardCategories());
```
[the remark this answers is not in the document]
Comment (thechechogomez) [reply libs/employer/feature/dashboard/src/lib/components/dashboard/dashboard.component.ts]: @derekdiep42 Fixed!

**Review by derekdiep42 — APPROVED** [review]
LGTM!

**Review by derekdiep42 — APPROVED** [review]

**Pull request merged**